app/src/controller/UserController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Src\controller;
+
+use Src\router\Request;
+use Src\router\Response;
+use Src\service\UserService;
+use Src\utils\JwtUtil;
+
+class UserController
+{
+    /**
+     * Moving this into a static way
+     */
+    // private Request $request;
+    // private Response $response;
+
+    // public function __construct(Request $request, Response $response)
+    // {
+    //     $this->request = $request;
+    //     $this->response = $response;
+    // }
+
+    public static function logIn(Request $req, Response $res)
+    {
+        $body = $req->getJson();
+        $userService = new UserService();
+        $user = $userService->getUserByEmail($body->email);
+
+        if (hash_equals(hash('sha1', $body->password), $user[0]['password'])) {
```

Comment:
@jjsimonAB use the password function from PHP , better solution than this https://www.php.net/manual/en/function.password-hash.php